Gemfile

```diff
@@ -20,7 +20,7 @@ gem "rails", "4.2.1"
 gem "recipient_interceptor"
 gem "refills"
 gem "sass-rails", "~> 5.0"
-gem "simple_form"
+# gem "simple_form"
```

Comment:
Leave it for now. We can decide about that later. Is there anything apart from this that is blocking this PR from being merged?